### PR TITLE
Fix CPU Profiler regression

### PR DESCRIPTION
**Pull request by Nekotekina**
Incorrectly tested exit flag after refactoring.

### CODE REVIEW

File: rpcs3/Emu/Cell/SPURecompiler.cpp [hunk 1]
```diff
@@ -539,6 +540,111 @@ void spu_cache::initialize()
...
+							sk = false;
+						}
+						else
+						{
+							dump += ' ';
+						}
+
+						dump += ' ';
+
+
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/SPURecompiler.cpp:630]: nani

File: rpcs3/Emu/Cell/SPURecompiler.cpp [hunk 2]
```diff
@@ -539,6 +540,111 @@ void spu_cache::initialize()
 	if ((g_cfg.core.spu_decoder == spu_decoder_type::asmjit || g_cfg.core.spu_decoder == spu_decoder_type::llvm) && !func_list.empty())
 	{
 		spu_log.success("SPU Runtime: Built %u functions.", func_list.size());
+
+		if (g_cfg.core.spu_debug)
+		{
+			std::string dump = "";
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/SPURecompiler.cpp:546]: isn't it empty by default?

File: rpcs3/Emu/Cell/SPURecompiler.cpp (hunk 1 above)
Comment (Nekotekina) [reply rpcs3/Emu/Cell/SPURecompiler.cpp]: Fixed

File: rpcs3/Emu/Cell/SPURecompiler.cpp (hunk 2 above)
Comment (Nekotekina) [reply rpcs3/Emu/Cell/SPURecompiler.cpp]: It is, just a remnant

**Pull request merged**